frontend/src/main/java/olumofe/raphael/frontend/CarController.java

```diff
@@ -0,0 +1,57 @@
+package olumofe.raphael.frontend;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.*;
+import starter.project.domain.Car;
+
+import java.util.*;
+
+@RestController
+@Component
+public class CarController {
+
+
+    private List<Car> carNewList = new ArrayList<>();
+
+    @GetMapping(path ="/car/{carId}", consumes = {"application/json"})
+    @ResponseBody
+    public Car getCars(@PathVariable("carId") int carId) {
+
+        for (Car car : carNewList) {
+            if (car.getId() == (carId)) {
```

Comment:
This logic is fine, the only thing is I don't think carNewList will ever have objects in it? So functionally the logic is fine but we will never get a car back. Perhaps give carNewList some initial values so we can see it working.

Reply:
Ive  not created a Posted method, and ive tested it to ensure that it works